RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift

```diff
@@ -30,9 +30,10 @@ extension TransactionReview.State {
 			viewControlState: viewControlState,
 			rawTransaction: displayMode.rawTransaction,
 			showApprovalSlider: reviewedTransaction != nil,
-			canApproveTX: canApproveTX && reviewedTransaction?.feePayingValidation == .valid,
+			canApproveTX: canApproveTX && reviewedTransaction?.feePayingValidation.wrappedValue == .valid,
 			sliderResetDate: sliderResetDate,
-			canToggleViewMode: reviewedTransaction != nil && reviewedTransaction?.transaction != .nonConforming
+			canToggleViewMode: reviewedTransaction != nil && reviewedTransaction?.transaction != .nonConforming,
+			viewRawTransactionButtonState: reviewedTransaction?.feePayer.isSuccess == true ? .enabled : .disabled
```

Comment:
Disable toggling to view raw transaction until the fee payer status is determined.